docs/DeveloperGuide.md

```diff
@@ -329,7 +316,27 @@ The implemented find by class code mechanism is facilitated by `FindByClassCode`
 
 #### Implementation
 
-The implemented Activity label is facilitated by `ClassCode`. It repurposes `AddressBook` `Tag` to Activity, tied to each person.
+The implemented status label is facilitated by `Activity` attribute. This label is an additional attribute for each person within the application
+and is implemented as a separate package within the `activity` Package. A `person` will have a Set of `activity` as an attribute.
+
+#### Design considerations:
+
+**Aspect: Abstracting `activity` attribute**
+
+* **Alternative 1 (current choice):** Abstracted class.
+    * Pros:
+        * Higher Level of abstraction
+        * Changes can be made easily from this class
+        * Ease of accommodating how `activity` attribute can be implemented and added in to the `Person` class as a `Set`
+    * Cons:
+        * Existing layers of abstraction and tangled dependencies make introducing a new attribute for the base Person model difficult
```

Comment:
Definitely true. This gets better with time as we get more used to working with the code base.